benchmarks/build-system-cost/main.go

```
@@ -0,0 +1,679 @@
+			ModuleSHA256:     fileDigest(filepath.Join(root, "MODULE.bazel")),
+			ModuleLockSHA256: fileDigest(filepath.Join(root, "MODULE.bazel.lock")),
+		},
+	}, nil
+}
+
+func measureTrial(ctx context.Context, root, output string, files []string, index int) (trialResult, error) {
+	trialRoot, err := os.MkdirTemp("", fmt.Sprintf("kivgraph-build-cost-%02d-", index))
+	if err != nil {
+		return trialResult{}, err
+	}
+	defer os.RemoveAll(trialRoot)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)